invenio_communities/communities/services/service.py

```diff
@@ -714,6 +714,30 @@ def on_relation_update(self, identity, record_type, records_info, notif_time):
 
         return True
 
+    @unit_of_work()
+    def update_parent_community(self, identity, id_, parent_id, uow=None):
```

Comment:
minor/question: isn't it possible to go via the usual `update` method (similar to how the `theme` is set?). In general we should try to avoid adding new service methods.